src/databricks/labs/ucx/recon/workflows.py

```diff
@@ -0,0 +1,15 @@
+from databricks.labs.ucx.contexts.workflow_task import RuntimeContext
+from databricks.labs.ucx.framework.tasks import Workflow, job_task
+
+
+class MigrationRecon(Workflow):
+    def __init__(self):
+        super().__init__('recon-migration-result')
```

Comment:
```suggestion
        super().__init__('data-reconciliation')
```
or something like it